frontend/assets/lang/pl.json

```
@@ -15,5 +15,48 @@
         "ComboboxField": "Wybierz kierunek",
         "ComboboxDepartment": "Wybierz wydział",
         "ComboboxSpecialization": "Wybierz specjalizację"
+    },
+    "SignIn":{
+        "Header": "Zaloguj się na swoje konto",
+        "Remember": "Zapamiętaj mnie",
+        "ForgotPassword": "Nie pamiętasz hasła?"
+    },
+    "Registration":{
+        "Header": "Rejstracja",
+        "FirstName": "Imię",
+        "LastName": "Nazwisko"
+    },
+    "AddCompany": {
+        "Header": "Zgłoś swoja firmę",
+        "HeaderInfo": "Po wypełnieniu formularza, twoje zgłoszenie zostanie przekazane moderatorowi do weryfikacji. ",
+        "CompanyName": "Nazwa firmy",
+        "Description": "Opis",
+        "DescriptionInfo": "Napisz coś o swojej firmie aby ułatwić wybór",
+        "StreetAddress": "Ulica",
+        "City": "Miasto",
+        "Voivodeship": "Województwo",
+        "PostalCode": "Kod pocztowy"
+    },
+    "ForgotPassword":{
+        "Header": "Podaj adres email, na którym założyłeś konto, wyślemy link resetujacy hasło",
```

```suggestion
        "Header": "Podaj adres e-mail, na którym zostało założone konto, wyślemy link resetujący hasło",
```

But we can think of something better.